crates/octez/src/async/bootstrap.rs

```diff
@@ -91,16 +95,127 @@ impl From<&BootstrapAccounts> for Value {
     }
 }
 
+#[serde_with::serde_as]
+#[serde_with::skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct BootstrapContract {
+    /// Smart contract code in micheline format as a `serde_json::Value` instance.
+    script: Value,
+    #[serde(rename = "amount")]
+    #[serde_as(as = "serde_with::DisplayFromStr")]
+    amount_mutez: u64,
+    hash: Option<ContractKt1Hash>,
```

Comment:
could you add a doc string comment to `hash` like how its explained in the Pr desc?

Reply:
Updated